v2/i18n/icons.go

```diff
@@ -0,0 +1,115 @@
+
+// Google Play Store icon links are hosted via frontend repo: https://github.com/ag5/js-monorepo
+func generateGooglePlayStoreIconLink(lang string) string {
+	return "https://cdn.ag5.com/mail-assets/mobile/googleplay/" + lang + ".png"
+}
+
+var googlePlayStoreLink = map[language.Tag]string{
+	language.Arabic:               generateGooglePlayStoreIconLink("ar"),    // ar
+	language.ModernStandardArabic: generateGooglePlayStoreIconLink("ar"),    // ar-001
+	language.Czech:                generateGooglePlayStoreIconLink("cs"),    // cs
+	language.Danish:               generateGooglePlayStoreIconLink("da"),    // da
+	language.German:               generateGooglePlayStoreIconLink("de"),    // de
+	language.English:              generateGooglePlayStoreIconLink("en"),    // en
+	language.AmericanEnglish:      generateGooglePlayStoreIconLink("en"),    // en-US
+	language.BritishEnglish:       generateGooglePlayStoreIconLink("en"),    // en-GB
+	language.Spanish:              generateGooglePlayStoreIconLink("es"),    // es
+	language.EuropeanSpanish:      generateGooglePlayStoreIconLink("es"),    // es-ES
+	language.LatinAmericanSpanish: generateGooglePlayStoreIconLink("es"),    // es-419
+	language.Finnish:              generateGooglePlayStoreIconLink("fi"),    // fi
+	language.French:               generateGooglePlayStoreIconLink("fr"),    // fr
+	language.Hebrew:               generateGooglePlayStoreIconLink("he"),    // he
+	language.Indonesian:           generateGooglePlayStoreIconLink("id"),    // id
+	language.Italian:              generateGooglePlayStoreIconLink("it"),    // it
+	language.Japanese:             generateGooglePlayStoreIconLink("ja"),    // ja
+	language.Korean:               generateGooglePlayStoreIconLink("ko"),    // ko
+	language.Dutch:                generateGooglePlayStoreIconLink("nl"),    // nl
+	language.Norwegian:            generateGooglePlayStoreIconLink("no"),    // no
+	language.Polish:               generateGooglePlayStoreIconLink("pl"),    // pl
+	language.Portuguese:           generateGooglePlayStoreIconLink("pt"),    // pt
+	language.BrazilianPortuguese:  generateGooglePlayStoreIconLink("pt-br"), // pt-BR
+	language.EuropeanPortuguese:   generateGooglePlayStoreIconLink("pt"),    // pt-PT
+	language.Russian:              generateGooglePlayStoreIconLink("ru"),    // ru
+	language.Swedish:              generateGooglePlayStoreIconLink("sv"),    // sv
+	language.Thai:                 generateGooglePlayStoreIconLink("th"),    // th
+	language.Turkish:              generateGooglePlayStoreIconLink("tr"),    // tr
+	language.Ukrainian:            generateGooglePlayStoreIconLink("uk"),    // uk
+	language.Vietnamese:           generateGooglePlayStoreIconLink("vi"),    // vi
+	language.Chinese:              generateGooglePlayStoreIconLink("zh"),    // zh
+	language.SimplifiedChinese:    generateGooglePlayStoreIconLink("zh"),    // zh-Hans
+	language.TraditionalChinese:   generateGooglePlayStoreIconLink("zh"),    // zh-Hant
```

Comment:
If you want to get the code a bit smaller, I'd specify the mapping once from `language.Tag` to code (e.g. "zh"), assuming that this mapping is always the same. And then use that in both of the icon functions (then you also only have to call `generateGooglePlaystoreIconLink` once)

Reply:
Good point, done.